src/AccordionItemHeading/AccordionItemHeading.tsx

```diff
@@ -12,6 +12,18 @@ type AccordionItemHeadingProps = React.HTMLAttributes<HTMLDivElement> & {
 
 type AccordionItemHeadingState = {};
 
+const getClosest = (el, selector) => {
```

Comment:
Could this please be moved into a 'helper' module like `src/helpers/getClosestElement`?